css/main.css

```diff
@@ -34,6 +34,60 @@ label.dynamic-info {
 	font-style: italic;
 }
 
+input[type='number'] {
+	max-width: 7em;
+	/* Remove spinner controls from Firefox */
+	-moz-appearance: textfield;
+}
+/* Re-applies the controls on :hover and :focus
+to get the same behavior as on chrom* */
+input[type='number']:hover,
+input[type='number']:focus {
+	-moz-appearance: number-input;
```

Comment:
Don't set these in `main.css`, this is a cross-browser adjustment. Create a specific file for them.